index.html

```diff
@@ -42,53 +56,36 @@
         // images to expose the images underneath. Note that all images are
         // expected to have the same position and size.
         const divideImages = () => {
-            const overlay1 = viewer.world.getItemAt(1);
-            const overlay2 = viewer.world.getItemAt(2);
-            const clipPos = overlay1.viewerElementToImageCoordinates(mousePos);
+            // Get the clip point and clamp it to within the image bounds.
+            const image = viewer.world.getItemAt(0)
+            const clipPos = image.viewerElementToImageCoordinates(mousePos);
+            const size = image.getContentSize();
+            clipPos.x = Math.max(0, Math.min(clipPos.x, size.x));
+            clipPos.y = Math.max(0, Math.min(clipPos.y, size.y));
 
-            // Clamp the clip point to within the image bounds.
-            const size = overlay1.getContentSize();
-            const xClip = Math.max(0, Math.min(clipPos.x, size.x));
-            const yClip = Math.max(0, Math.min(clipPos.y, size.y));
+            // Set the clip for each image.
+            let previousVisibleImages = 0;
+            for (let i = 0; i < viewer.world.getItemCount(); ++i) {
+                const image = viewer.world.getItemAt(i);
+                if (!image.getOpacity()) {
+                    continue;
+                }
+                // Determine the quadrants to be clipped by how many visible
+                // images are underneath this one.
+                const xClip = previousVisibleImages & 1 ? clipPos.x : 0;
+                const yClip = previousVisibleImages & 2 ? clipPos.y : 0;
```

Comment:
This is some bitwise magic so that the images are cropped, in order, as: full, right half, bottom half, bottom-right quarter (currently unused).